community/modules/scheduler/schedmd-slurm-gcp-v6-controller/main.tf

```diff
@@ -29,6 +29,8 @@ locals {
 }
 
 data "google_compute_default_service_account" "default" {
+  count = var.service_account_email == null ? 1 : 0
```

Comment:
Or empty string, to match behavior of `coalesce`.

Reply:
Changed it to check for null or empty string.